docs/guides/migrate-from-buidler.md

```diff
@@ -0,0 +1,145 @@
+# Migrating from Buidler
+
+Hardhat is the new and evolved version of Buidler.
+
+This guide will teach you how to migrate your project from Buidler into Hardhat.
+
+## Installing the Hardhat packages
+
+The first thing you need to do to migrate your project, is installing th new npm packages.
+
+The package `@nomiclabs/buidler` is now `hardhat`. The plugins, which used to have packages like `@nomiclabs/buidler-<name>`,
+are now `@nomiclabs/hardhat-<name>`.
+
+For example, if you were using `@nomiclabs/buidler` and `@nomiclabs/buidler-ethers`, you need to run:
+
+```
+npm install --save-dev hardhat @nomiclabs/hardhat-ethers
+```
+
+If you were using a global installation of Buidler, you also need to install Hardhat locally.
+Global installations of Hardhat are not supported.
+
+## Adapting your config
+
+You can use your Buidler config in Hardhat mostly unchanged. All you need to do is following these steps.
+
+### Renaming your config file
+
+First, you need to rename your config file from `buidler.config.js` to `hardhat.config.js`. Or,
+if you are using TypeScript, from `buidler.config.ts` to `hardhat.config.ts`.
+
+### Changing how your plugins are loaded
+
+Then, you have to change how your plugins are loaded. Instead of using the `usePlugin` function, you just
+have to require/import their npm package. For example, if you had
+
+```js
+usePlugin("@nomiclabs/buidler-ethers");
+```
+
+you need to replace it with
+
+```js
+require("@nomiclabs/buidler-ethers");
+```
+
+Or, if you are using TypeScript, with
+
+```ts
+import "@nomiclabs/buidler-ethers";
+```
+
```

Comment:
This might be obvious, but you also need to remove the `usePlugin` import in typescript configs.